src/main/java/net/runelite/rsb/wrappers/client_wrapper/RSClient.java

```diff
@@ -140,6 +147,30 @@ public Widget getScriptDotWidget() {
         return convertResult(super.getScriptDotWidget());
     }
 
+    @Override
+    public int getCameraYawTarget() {
+        return 0;
+    }
+
+    @Override
+    public int getCameraPitchTarget() {
+        return 0;
+    }
+
+    @Override
+    public void setCameraPitchTarget(int cameraPitchTarget) {}
+
+    @Override
+    @Deprecated
+    public RenderOverview getRenderOverview() {
+        return null;
+    }
+
+    @Override
+    public WorldMap getWorldMap() {
```

Comment:
Why leave these overrides in when the wrapper has proper implementations?

Reply:
oversight, thanks for alerting me